dds/Cargo.toml

```diff
@@ -16,9 +16,10 @@ keywords = ["dds", "rtps", "middleware", "network", "udp"]
 categories = ["api-bindings", "network-programming"]
 
 [dependencies]
-dust_dds_derive = { path = "../dds_derive", version = "0.3" }
+dust_dds_derive = { path = "../dds_derive", version = "0.4" }

```

Comment:
version should disappear as for dds_gen

Reply:
I think here it has to stay because we re-export the macros so when someone uses our crate this has to come from the actual crates.io instead of from only the repo. The dds_gen is only used for tests internally so there it is ok without version.